julia/core.py

```diff
@@ -140,8 +140,6 @@ class JuliaImporter(object):
 
     # find_module was deprecated in v3.4
     def find_module(self, fullname, path=None):
-        if path is None:
```

Comment:
It looks like this code was added in https://github.com/JuliaPy/pyjulia/commit/fa157b19e6fcd6fdcdd2f0e7584e6d9c75cc091b and carried around as-is.

Probably just a remnant of experimental code?